assets/style.css

```diff
@@ -1,54 +1,50 @@
 /* CSS styling elements go here */
 :root {
-    --soft-white: #fcfdf4;
-    /* no seriously, daylight has a known color, this should be close */
-    --daylight: #fcfcff;
-    --lavender: #e4e0fd;
-    --lime: #5af7a2;
-    --coral: #fbbdbd;
-    --straw: #fff8ea;
-    --orange-smoothie: #ffccae;
-    --slate-blue: #a9b3cc;
-    --deep-scarlet: #e80040;
-    --primary-green: #00d1b2;
-    --pale-indigo: #e5e7fa;
+  --soft-white: #fcfdf4;
+  /* no seriously, daylight has a known color, this should be close */
```

Comment:
remove commented code

Reply:
one of those is in active use, the others are used for other thing. I can comment out the two that aren't, but why?